modules/reitit-frontend/src/reitit/frontend.cljs

```diff
@@ -37,12 +37,17 @@
                    coercion/coerce!)]
      (if-let [match (r/match-by-path router (.getPath uri))]
        (let [q (query-params uri)
-             match (assoc match :query-params q)
+             fragment (when (.hasFragment uri)
```

Comment:
If we want to have a fragment value as a string always, this condition could be dropped.
`hasFragement` just checks if the parsed value is a non-empty string; there is no different between URL without `#` and one with a fragment but nothing after `#`.